### PR TITLE
Add Various Symbols To Futures Database (Part 2 / 11)

### PR DESCRIPTION
**Pull request by gsalaz98**
#### Description

Adds The Following Symbols To Futures Database: 6Z, A0D, A0F, A1L, A1M, A1R, A32, A3G, A7E, A7I, A7Q, A8J, A8K, A8O, A91, A9N, AA6, AA8, ABS, ABT, AC0, ACD, AD0, ADB, AE5

**Note to reviewer:** symbol ACD has two potential values for tick size. I may have the wrong minimum price variation for the symbol in `symbol-properties-database.csv`. You can find more information about that symbol here: https://www.cmegroup.com/trading/fx/g10/australian-dollar-canadian-dollar_contract_specifications.html
#### Related Issue

#2991
#### Motivation and Context

Ongoing completion efforts of the futures database
#### Requires Documentation Change

No
#### How Has This Been Tested?

New unit tests
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Data/symbol-properties/symbol-properties-database.csv [hunk 1]
```diff
@@ -103,8 +103,34 @@ usa,6N,future,,USD,100000,0.0001,1
 usa,6R,future,,USD,2500000,0.000005,1
 usa,6L,future,,USD,100000,0.00005,1
 usa,6M,future,,USD,500000,0.00001,1
+usa,6Z,future,,USD,500000,0.000025,1
+# TODO: verify if ACD is accurate
```
Comment (Jay-Jay-D) [inline Data/symbol-properties/symbol-properties-database.csv]: What is needed to verify this?

Comment (gsalaz98) [reply Data/symbol-properties/symbol-properties-database.csv]: I'd like a second opinion regarding that symbol's minimum-price-variation. The CME website lists two potential variations, and I didn't know which one to choose.

I put a link to the symbol's specs in the original commit message. Check it out and let me know what your thoughts on it are :+1:

Comment (gsalaz98) [reply Data/symbol-properties/symbol-properties-database.csv]: Seems I've put the wrong currency for that symbol and the minimum-price-variation is wrong as well. Just found a page with all the details I need. Fix will be coming out shortly

**Comment by jaredbroad** [comment]
Nice work @gsalaz98

**Pull request merged**